src/styles/card.scss

```
@@ -0,0 +1,95 @@
+html {
+  --color-for-btn: #00acdc;
+  --color-for-bgn: #fff;
```

```suggestion
  --default-background: #fff;
```
It is clear that it is color because of hex and usage as value of `color` propery